fuzzers/aflplusplus_qemu/fuzzer.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Integration code for AFLplusplus fuzzer."""
+
+import os
+import subprocess
+
+from fuzzers.aflplusplus import fuzzer as aflplusplus_fuzzer
+
+# OUT environment variable is the location of build directory (default is /out).
+
+
+def build():
+    """Build benchmark."""
+    aflplusplus_fuzzer.build('qemu')
+
+
+def fuzz(input_corpus, output_corpus, target_binary):
+    """Run fuzzer."""
+    # get LLVMFuzzerTestOneInput address
+    nm_proc = subprocess.run([
+        'sh', '-c',
+        'nm \'' + target_binary + '\' | grep \'T afl_qemu_driver_stdin_input\''
+    ],
+                             stdout=subprocess.PIPE,
+                             check=True)
+    target_func = "0x" + nm_proc.stdout.split()[0].decode("utf-8")
```

Comment:
Impressive, was this hard to figure out?

Reply:
we use that in our test framework, but with shell script not python :)
Andrea did this.